builtin/providers/aws/resource_aws_elasticache_replication_group_test.go

```diff
@@ -235,7 +235,7 @@ func TestResourceAWSElastiCacheReplicationGroupIdValidation(t *testing.T) {
 			ErrCount: 1,
 		},
 		{
-			Value:    randomString(17),
+			Value:    randomString(21),
```

Comment:
I want to check that when we create a string > 20 that we fail the validation func - I guess i could add another test if you want to prove both work?